quixstreams/rowconsumer.py

```diff
@@ -128,7 +128,7 @@ def poll_row(self, timeout: float = None) -> Union[Row, List[Row], None]:
         if msg is None:
             return
 
-        topic_name, partition, offset = msg.topic(), msg.partition(), msg.offset()
+        topic_name = msg.topic()
```

Comment:
@daniil-quix Please advise if this change is safe. I believe `partition` and `offset` methods have no side effects so it's safe to remove.

Reply:
There're no side effects, they simply return the partition number and the offset